dali/operators/reader/loader/video_loader.cc

```diff
@@ -533,6 +533,11 @@ void VideoLoader::read_file() {
     bool is_first_frame = true;
     bool key = false;
     bool seek_must_succeed = false;
+    // how many key frames following the last requested frames we need to see so far
+    int key_frames_count = 0;
+    // how many key frames following the last requested frames we need to see before we stop
+    // feeding the decoder
+    const int key_frames_treshold = 1;
```

Comment:
IMO we want to see two, so I would probably write here 2, and use greater equal, but it's a nitpick:
```suggestion
    const int key_frames_treshold = 2;
```